.github/workflows/pre-commit.yml

```
@@ -0,0 +1,14 @@
+name: pre-commit
+
+on:
+  pull_request:
+  push:
+    branches: [master]
```

```suggestion
    branches: master
```
Sorry this is so nitty, but I think brackets are not needed

---

are you sure? I took this from here https://github.com/pre-commit/action

---

also, my linter says "Incorrect type. Expected "array"." if I remove the brackets

---

You're probably right on both counts, I was comparing with the config used by dask. Ignore me 😄

---

:laughing: 

have had a look at dask's repo, I gather it's fine to write to both ways